### PR TITLE
[FIX] website_sale: Added base_url to schema.org microdata tags

### PR DESCRIPTION
**Pull request by stcc-odoo**
Steps to reproduce the bug:
- install eCommerce module with assets
- go to any product page
- validate the source code with https://validator.schema.org/

Notice that the 'image' and 'url' microdata tags have no base url,
so the engine assumes http://schema.org/ to be the base url. This
makes certain ad trackers such as facebook pixel unable to detect
the product being sold.

This commit adds the base_url field to both tags so that they are
correctly traced.

opw-2830058

### CODE REVIEW

File: addons/website_sale/views/templates.xml [hunk 1]
```diff
@@ -1744,7 +1746,7 @@
                             <div t-if="product_image._name == 'product.image' and product_image.embed_code" class="d-flex align-items-center justify-content-center h-100 embed-responsive embed-responsive-16by9">
                                 <t t-raw="product_image.embed_code"/>
                             </div>
-                            <div  t-else="" t-field="product_image.image_1920" class="d-flex align-items-center justify-content-center h-100" t-options='{"widget": "image", "preview_image": "image_1024", "class": "product_detail_img mh-100", "alt-field": "name", "zoom": product_image.can_image_1024_be_zoomed and "image_1920", "itemprop": "image"}'/>
+                            <div  t-else="" t-field="product_image.image_1920" class="d-flex align-items-center justify-content-center h-100" t-options='{"widget": "image", "preview_image": "image_1024", "class": "product_detail_img mh-100", "alt-field": "name", "zoom": product_image.can_image_1024_be_zoomed and "image_1920"}'/>
```
Comment (Williambraecky) [inline addons/website_sale/views/templates.xml:1749]: Is this change really necessary?

**Review by Williambraecky — COMMENTED** [review]
Thanks for the fix!
One small concern :)

File: addons/website_sale/views/templates.xml (hunk 1 above)
Comment (stcc-odoo) [reply addons/website_sale/views/templates.xml:1749]: I'm pretty sure it is. If it is not removed, then there will be two DOM elements with the attribute "itemprop"="image" (this one + the one I added) and from testing I noticed that this one will be picked by the schema validator. Even if it wasn't picked, its value is still wrong, since it doesn't contain the base_url.

Comment (Williambraecky) [reply addons/website_sale/views/templates.xml:1749]: Then it was already broken since all images would have had the itemprop attribute if there are multiple images to display. 😬

**Review by Williambraecky — APPROVED** [review]
Thanks for the fix!
robodoo r+
